src/articles/FindArticles.js

```diff
@@ -11,21 +11,14 @@ import ExtensionMessage from "./ExtensionMessage";
 import LocalStorage from "../assorted/LocalStorage";
 
 import ShowLinkRecommendationsIfNoArticles from "./ShowLinkRecommendationsIfNoArticles";
-import { useLocation } from "react-router-dom";
 import { APIContext } from "../contexts/APIContext";
 import useExtensionCommunication from "../hooks/useExtensionCommunication";
 import {
   getPixelsFromScrollBarToEnd,
   isScrollable,
 } from "../utils/misc/getScrollLocation";
-// A custom hook that builds on useLocation to parse
-// the query string for you.
-function useQuery() {
-  return new URLSearchParams(useLocation().search);
-}
 
-export default function FindArticles() {
-  const searchQuery = useQuery().get("search");
+export default function FindArticles({ contentSearch, searchQuery }) {
```

Comment:
This new implementation with `contentSearch` and `searchQuery` is hard to understand. 

Given that you want to test already on Wednesday, if we can't improve it till then, I propose that we deploy like this, but you open an issue that says that we have to refactor it, and we do it at a later time.